stl/src/winapisupp.cpp

```diff
@@ -4,18 +4,16 @@
 // clang-format off
 // Prevent clang-format from reordering <AppModel.h> before <Windows.h>
 #include <Windows.h>
-#include <AppModel.h>

```

Comment:
I believe that `<AppModel.h>` must be restored - this file is referring to the constant `APPMODEL_ERROR_NO_PACKAGE` [which appears](https://docs.microsoft.com/en-us/windows/win32/api/appmodel/nf-appmodel-getcurrentpackagepath) to be provided by that header. Note that the "app" flavor is one of the builds that we still need to port to GitHub, so a green checkmark here may conceal a build failure that we'd discover when porting to the MSVC-internal legacy build system.